wfe2/wfe.go

```diff
@@ -353,12 +358,15 @@ func (wfe *WebFrontEndImpl) Handler() http.Handler {
 	wfe.HandleFunc(m, directoryPath, wfe.Directory, "GET", "POST")
 	wfe.HandleFunc(m, newNoncePath, wfe.Nonce, "GET", "POST")
 	// POST-as-GETable ACME endpoints
-	// TODO(@cpu): After November 1st, 2019 support for "GET" to the following
-	// endpoints will be removed, leaving only POST-as-GET support.
-	wfe.HandleFunc(m, orderPath, wfe.GetOrder, "GET", "POST")
-	wfe.HandleFunc(m, authzv2Path, wfe.AuthorizationV2, "GET", "POST")
-	wfe.HandleFunc(m, challengev2Path, wfe.ChallengeV2, "GET", "POST")
-	wfe.HandleFunc(m, certPath, wfe.Certificate, "GET", "POST")
+	wfe.HandleFunc(m, orderPath, wfe.GetOrder, "POST")
+	wfe.HandleFunc(m, authzv2Path, wfe.AuthorizationV2, "POST")
+	wfe.HandleFunc(m, challengev2Path, wfe.ChallengeV2, "POST")
+	wfe.HandleFunc(m, certPath, wfe.Certificate, "POST")
```

Comment:
Apologies, the TODO you saw should have been updated to say **November 1st, 2020**. We pushed back the production deprecation of unauthenticated GETs.

That means you'll have to revert this diff and restore the "GET" method for each of these handlers. We can't remove support until after Nov 2020.

You should be able to revert your diff to the unit tests you changed in `wfe2/wfe_test.go` at the same time as undoing the changes here. Those tests will function as-written if "GET" method is still allowed for these endpoints in a configuration without the `features.MandatoryPOSTAsGet` feature flag.

Reply:
should be okay now.

Reply:
Thanks!